src/commands.c

```diff
@@ -598,19 +585,10 @@ parse_command_from_file (const char *cmd)
 
 /* ========================= COMMAND TABLE ========================== */
 
-#ifdef USE_WEBKIT2
 #if WEBKIT_CHECK_VERSION (1, 11, 4)
 #define HAVE_PLUGIN_API
 #endif
 #define HAVE_SECURITY
```

Comment:
This and the plugin API seem to always be true, so simplifying based on that is possible.